cmd/oci-runtime-tool/generate.go

```diff
@@ -22,8 +22,6 @@ var generateFlags = []cli.Flag{
 	cli.StringFlag{Name: "arch", Value: runtime.GOARCH, Usage: "architecture the container is created for"},
 	cli.StringSliceFlag{Name: "args", Usage: "command to run in the container"},
 	cli.StringSliceFlag{Name: "bind", Usage: "bind mount directories src:dest[:options...]"},
-	cli.StringSliceFlag{Name: "cap-add", Usage: "add Linux capabilities"},
-	cli.StringSliceFlag{Name: "cap-drop", Usage: "drop Linux capabilities"},
```

Comment:
We can use these flags to add/drop from all the 5 sets. We can add other more specific flags like cap-bounding-add/cap-bounding-drop after this PR. Thanks!